web/book/src/language-features/README.md

```diff
@@ -3,16 +3,20 @@
 The pages of this section describe how PRQL handles various aspects of the
 language.
 
-| Feature          | Purpose                                                                 |
-| ---------------- | ----------------------------------------------------------------------- |
-| Coalesce         | [Handle potentially NULL values](./coalesce.md)                         |
-| Dates & times    | [Handle dates and times](./dates-and-times.md)                          |
-| Distinct         | [Equivalent of SQL `DISTINCT`](./distinct.md)                           |
-| Null handling    | [Handle `NULL` values](./null.md)                                       |
-| Ranges           | [Syntax for all forms of ranges](./ranges.md)                           |
-| Regex            | [Handle regular expressions](./regex.md)                                |
-| Strings          | [Rules for creating strings](./strings.md)                              |
-| S-strings        | [Insert SQL directly into a query with an S-string](./s-strings.md)     |
-| F-strings        | [Combine several column's data with F-strings](./f-strings.md)          |
-| Case             | [Create a new column based on the contents of other columns](./case.md) |
-| Target & Version | [Specify a target SQL engine and PRQL version](./target.md)             |
+<!-- we're linking to README.md files as index.html to work around https://github.com/rust-lang/mdBook/issues/984  by using "markdown_link_check_disable"-->
```

Comment:
I'd remove the `by using "markdown_link_check_disable` from that comment since it's not actually part of the workaround but just to make the link checker happy.

Reply:
I think the comment should be rewritten to say:

```
<!-- 
We use "markdown_link_check_disable" to work around https://github.com/rust-lang/mdBook/issues/984  
so we can link to "index.html" files without complaints from `markdown-link-check`
-->

Reply:
that's still wrong ... markdown_link_check_disable does not work around the mdbook bug ... the workaround is the change of the extension in the link

Reply:
Right again! (Actually, making the link checker happy is _another_ workaround...) How about...
```
<!-- 
To work around https://github.com/rust-lang/mdBook/issues/984 we link to "index.html" instead of "README.md"
files. To avoid complaints from `markdown-link-check` in the CI, we use "markdown-link-check-disable" 
-->
```